test/unit/vl-progress-bar-item.test.html

```diff
@@ -0,0 +1,64 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../../web-component-tester/browser.js"></script>
+
+  <script type="module" src="../../src/vl-progress-bar.js"></script>
+  <script type="module" src="../../src/vl-progress-bar-item.js"></script>
+</head>
+
+<body>
+  <test-fixture id="vl-progress-bar-fixture">
+    <template>
+      <vl-progress-bar>
+        <vl-progress-bar-item data-vl-active>Stap 1/2: Aanvraag</vl-progress-bar-item>
+        <vl-progress-bar-item>Stap 2/2: Bevestigen</vl-progress-bar-item>
+      </vl-progress-bar>
+    </template>
+  </test-fixture>
+
+  <script>
+    suite('<vl-progress-bar-item>', () => {
+      const should = chai.should();
+
+      test('de stap tekst kan opgevraagd worden', () => {
+        const progressBar = fixture('vl-progress-bar-fixture');
+        const progressBarItems = progressBar.querySelectorAll('vl-progress-bar-item');
+        assert.equal(progressBarItems[0].text, 'Stap 1/2: Aanvraag');
+        assert.equal(progressBarItems[1].text, 'Stap 2/2: Bevestigen');
+      });
+
+      test('de stap kan als actief gezet worden via een attribuut', () => {
+        const progressBar = fixture('vl-progress-bar-fixture');
+        const progressBarItems = progressBar.querySelectorAll('vl-progress-bar-item');
+        assert.isTrue(progressBarItems[0].active);
+        assert.isFalse(progressBarItems[1].active);
+        progressBarItems[0].removeAttribute('data-vl-active');
+        assert.isFalse(progressBarItems[0].active);
+        assert.isFalse(progressBarItems[1].active);
+        progressBarItems[0].setAttribute('data-vl-active', '');
+        assert.isTrue(progressBarItems[0].active);
+        assert.isFalse(progressBarItems[1].active);
+      });
+
+      test('de stap kan als actief gezet worden via de API', () => {
+        const progressBar = fixture('vl-progress-bar-fixture');
+        const progressBarItems = progressBar.querySelectorAll('vl-progress-bar-item');
+        assert.isTrue(progressBarItems[0].active);
+        assert.isFalse(progressBarItems[1].active);
+        progressBarItems[0].active = false;
+        assert.isFalse(progressBarItems[0].active);
+        assert.isFalse(progressBarItems[1].active);
+        progressBarItems[0].active = true;
+        assert.isTrue(progressBarItems[0].active);
+        assert.isFalse(progressBarItems[1].active);
+      });
+    });
+
```

Comment:
moet je hier ook niet testen dat de template er anders uit ziet naargelang de step actief is of niet?

Reply:
Die test zit bij de testen van de vl-progress-bar.